testdata/p4_16_samples/issue383-bmv2.p4

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
This test is identical to bitvec_hdr.p4. Why have two identical tests? Keep just this one, and delete the other one.
But please read the comments I made for that test.

Reply:
I wanted to compile issue383-bmv2.p4 using p4c-bm2-ss and compile bitvec_hdr.p4 with p4test.  This is why I created two identical files.  If another way exists to compile one file, two time, with the two compilers, I can make the change.

Reply:
each bmv2 file is also compiled by p4test